packages/image_picker/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -368,6 +393,26 @@ private void handleChooseImageResult(int resultCode, Intent data) {
     finishWithSuccess(null);
   }
 
+  private void handleChooseMultiImageResult(int resultCode, Intent data) {
```

Comment:
naming the intent data is confusing, maybe just "Intent intent"?

Reply:
Done.